src/test/java/org/mdbenefits/app/testutils/Page.java

```diff
@@ -339,15 +339,18 @@ public WebElement findAccordionByButtonText(String buttonText) {
                 .orElse(null);
     }
 
-    public boolean elementDoesNotExistById(String id) {
+    public boolean elementExistsById(String id) {
```

Comment:
I created this method and revamped the one below it so they played off of each other in a logical fashion.